src/functions/forecast/list-time-entries/handler.ts

```diff
@@ -58,6 +70,9 @@ const listTimeEntriesHandler: ValidatedEventAPIGatewayProxyEvent<any> = async ev
   
   const timeEntries = await listTimeEntries(api, {
     projectId: parseInt(event.queryStringParameters.projectId),
+    startDate: event.queryStringParameters.startDate ? event.queryStringParameters.startDate : undefined,
```

Comment:
I can't leave a comment on the appropriate line due to github limitations but on the first line of the listTimeEntriesHandler we need the optional chaining for !event?.queryStringParameters.projectId, perhaps the message could also be updated to Missing parameters rather than Invalid parameters